src/main/java/frc/robot/RobotContainer.java

```diff
@@ -65,7 +67,7 @@ public class RobotContainer {
   private final PS4Controller opController = new PS4Controller(1);
 
   private final RobotArmSubsystem arm = new RobotArmSubsystem();
-  private final RobotArmControl ControlArm = new RobotArmControl(arm);
+  private final RobotArmControl controlArm = new RobotArmControl(arm);
```

Comment:
Not a related change, but it can be left in. In the future try to separate out work into separate PRs

Reply:
Okay. I was kind of just updating everything that was throwing up errors so it wouldn't mess with the pull request.